src/OrleansServiceBus/OrleansServiceBus.csproj

```diff
@@ -78,6 +78,10 @@
       <Project>{0054DB14-2A92-4CC0-959E-A2C51F5E65D4}</Project>
       <Name>OrleansProviders</Name>
     </ProjectReference>
+    <ProjectReference Include="..\OrleansRuntime\OrleansRuntime.csproj">
+      <Project>{6FF2004C-CDF8-479C-BF27-C6BFE8EF93E0}</Project>
+      <Name>OrleansRuntime</Name>
+    </ProjectReference>
```

Comment:
What's the implication to clients using event hub streams? Not necessarily complaining about not adding the dependency, but if we do, we either need to add a dependency in the nuspec, or dynamically test whether we are running on the silo before assuming that the Orleans.Runtime DLL is deployed (which is a Silo dll).